server/controllers/asset_management/assets/reports/needed_inventory_scans.js

```diff
@@ -39,7 +39,6 @@ async function neededInventoryScansReport(req, res, next) {
       is_asset : scan.is_asset,
       scan_start_date : new Date(scan.start_date),
       scan_end_date : new Date(scan.end_date),
```

Comment:
I do not think you should remove this.   The purpose of the "reference_number" is to allow correlation of scans to some reference such as a purchase order, etc.   It is used when you create an "Needed Inventry Scan" and is saved in the 'required inventory scan".  It should be coming into this function via the database query in line 35. And it appears in the report itself, so deleting it here is a problem.  You should check the value of "scan" after the database query on line 35 and make sure reference_number is reasonable (or blank).  If it is coming in as NULL, you should probably make sure the database table has a default value (like an empty string) of that the assignment you removed from like 42 has a fallback/default value:     scan.reference_number || '',

Also, you should double-check to make sure we are using the "reference_number" correctly.  It occurs to me that we may have a logical issue:  It seems to me that each asset could have its own reference number (bought under different contracts, etc).  In that case 'reference_number' should be have a column of its own in the Required Inventory Report.